using_images/xpaas_images/sso.adoc

```diff
@@ -0,0 +1,166 @@
+
+* *_sso70-basic_* - SSO backed by a H2 database
+* *_sso70-mysql_* - SSO backed by a MySQL database
+* *_sso70-mysql-persistent_* - SSO backed by a persistent PostgreSQL database
+* *_sso70-postgresql_* - SSO backed by a MySQL database
+* *_sso70-postgresql-persistent_* - SSO backed by a persistent PostgreSQL database
+
+An SSO-enabled Red Hat JBoss Enterprise Application Platform (JBoss EAP) Image has been offered which enables users to deploy a JBoss EAP instace that can use SSO for authentication:
+
+* *_eap64-sso-s2i_* - SSO-enabled JBoss EAP
+
+== Differences Between the OpenShift Red Hat Single Sign-On xPaaS Application and Keycloak
+The OpenShift Red Hat Single Sign-On xPaaS application is built from Keycloak, a JBoss community project. There are some differences in functionalitity between the Red Hat Single Sign-On xPaaS Application and Keycloak:
+
+* This image is currently available as a Technical Preview for use only with SSO-enabled Red Hat JBoss Enterprise Application Platform (JBoss EAP) applications.
+* The Red Hat Single Sign-On xPaaS Technical Preview Application includes all of the functionality of Keycloak 1.8.1, with the addition of a .war deployment that automatically handles the SSO client configuration.
+
+== Versioning for xPaaS Images
+See the xPaaS part of the https://access.redhat.com/articles/2176281[OpenShift and Atomic Platform Tested Integrations page] for details about OpenShift image version compatibility.
+
+== Prerequisites for Deploying the Red Hat Single Sign-On xPaaS Image
+The following is a list of prerequisites for using the Red Hat Single Sign-On xPaaS image:
+
+. *Acquire Red Hat Subscriptions* - Ensure that you have the relevant subscriptions for OpenShift as well as a subscription for xPaaS Middleware.
+. *Install OpenShift* - Before using the OpenShift xPaaS images, you must have an OpenShift environment installed and configured:
+.. The https://docs.openshift.com/enterprise/3.1/install_config/install/quick_install.html[Quick Installation] method allows you to install OpenShift using an interactive CLI utility.
+.. The https://docs.openshift.com/enterprise/3.1/install_config/install/advanced_install.html[Advanced Installation] method allows you to install OpenShift using a reference configuration. This method is best suited for production environments.
+. Ensure the *DNS* has been configured. This is required for communication between JBoss EAP and SSO, and for the requisite redirection. See https://docs.openshift.com/enterprise/3.1/install_config/install/prerequisites.html#prereq-dns[DNS] for more information.
+. *Install and Deploy Docker Registry* - Install the Docker Registry and then ensure that the Docker Registry is deployed to locally manage images:
++
+----
+$ oadm registry --config=/etc/origin/master/admin.kubeconfig \
+--credentials=/etc/origin/master/openshift-registry.kubeconfig
```

Comment:
For consistency, indent the second line with 4 spaces:

`
$ oadm registry --config=/etc/origin/master/admin.kubeconfig \
    --credentials=/etc/origin/master/openshift-registry.kubeconfig
`